content/src/content/jcr_root/apps/core/wcm/components/experiencefragment/v1/experiencefragment/experiencefragment.html

```diff
@@ -0,0 +1,27 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe Systems Incorporated
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div class="cmp-experience-fragment">
+	<sly data-sly-test="${!properties.fragmentpath}">
+	<div data-sly-test="${(wcmmode.edit || wcmmode.preview)}"
+		class="cq-dd-experiencefragment cq-placeholder "
+		data-emptytext="${component.properties.jcr:title}${emptyTextAppend && ' - '}${emptyTextAppend}">
+	</div>
+	</sly>
+	<sly
```

Comment:
We just had a question come up on another project - can there be support for Template Editor XF inclusion and specify a css class to include? 

The XFs doesn't support setting a container level Class name. The XF Template does, but that would mean creating a template per XF need (not valid design). 

So if the author and/or template editor can specify a css class (or style system options) at this location, it will support more flexible experience capabilities.

Reply:
I see the Style system support, but nothing on the Author side to enable this.